templates/base.html.twig

```diff
@@ -40,40 +40,42 @@
   </head>
 
   <body class="{% block page_classes '' %} {{ app.user ? 'user-logged-in' : 'user-anonymous' }} {{ (app.user.nightMode ?? false) ? 'night-mode' }}">
-    {% block site_feedback %}
-      {{ include('_includes/site_feedback.html.twig', {forum: forum ?? null}, with_context=false) }}
-    {% endblock %}
-
-    {% block site_banner %}
-      {{ include('_includes/site_banner.html.twig', {forum: forum ?? null}, with_context=false) }}
-    {% endblock %}
-
-    {% block site_nav %}
-      {{ include('_includes/site_nav.html.twig', {forum: forum ?? null}, with_context=false) }}
-    {% endblock %}
-
-    {% block site_alerts %}
-      <div class="site-alerts">
-        {%- for type, notices in app.flashes -%}
-          {%- for notice in notices -%}
-            <div class="alert alert--{{ type }} alert--animate alert--dismissable" role="alert">
-              <p>{{ notice|trans }}</p>
-            </div>
+    <div id="vue-app">
```

Comment:
I thought with el: vue will replace all contents, but you have site_feedback, banner, nav and other stuff in there?
shouldn't this just be <div id="vue-app" /> ?

Reply:
It definitely could be moved over to the `site_main` block. I just figured there may one day be a need to have Vue components rendered outside of that section.

Reply:
I just didn’t understand why it’s not replacing the content. If it works - it’s fine as is.

Reply:
Yeah i think `el` just sets the scope for Vue to look for "custom elements" in. It doesn't replace contents.